tests/openvino/test_modeling.py

```diff
@@ -2169,6 +2169,7 @@ def test_compare_with_and_without_past_key_values(self):
 class OVModelForVisualCausalLMIntegrationTest(unittest.TestCase):
     SUPPORTED_ARCHITECTURES = ["llava"]
     SUPPORT_VIDEO = []
+    SUPPORT_AUDIO = []
```

Comment:
if this is empty, the model is not being tested for audio no ?

Reply:
it is initailly empty, phi4mm is the first model that cover audio as input